server/src/modules/boards/boards.service.ts

```diff
@@ -67,6 +69,13 @@ export class BoardsService {
     boardId: string,
   ): Promise<BoardResponseObject> {
     const deletedBoard = await this.deleteBoardById(boardId);
+    for (const slide of deletedBoard.slides) {
```

Comment:
isnt it better to perform deleting logic in the stats service and call the method once implemented?